app/controllers/users/charity_position.js

```diff
@@ -0,0 +1,90 @@
+import Ember from "ember";
+import AsyncMixin, { ERROR_STRATEGIES } from "stock/mixins/async";
+
+export default Ember.Controller.extend(AsyncMixin, {
+  organisationService: Ember.inject.service(),
+  organisationsUserService: Ember.inject.service(),
+  i18n: Ember.inject.service(),
+
+  isInvalidPreferredNumber: Ember.computed(
+    "model.preferredContactNumber",
+    function() {
+      return (
+        !this.get("model.preferredContactNumber") ||
+        this.get("model.preferredContactNumber").length < 8
+      );
+    }
+  ),
+
+  isInvalidOrganisation: Ember.computed("organisation", function() {
+    return !this.get("model.organisation.nameEn");
+  }),
+
+  actions: {
+    /**
+     * Open's the organisation overlay.
+     * On change of organisation, a check is made
+     * if organisations_users record already exists for the user, then depending on that URL is changed
+     * which in-turn loads a model.
+     */
+    async serchOrganisation() {
+      const organisation = await this.get(
+        "organisationService"
+      ).userPickOrganisation();
+      this.set("organisation", organisation);
+
+      const organisationUser = this.get(
+        "organisationsUserService"
+      ).getOrganisationUser(organisation.get("id"), this.get("user_id"));
+
+      if (organisationUser) {
+        this.transitionToRoute(
+          `/users/${this.get(
+            "user_id"
+          )}/charity_position?id=${organisationUser.get("id")}`
+        );
```

Comment:
anyone else thinking we should remove that automatic prettifier ? I feel we should be allowed to pass in a long string to a `transition` method.

cc @bharat619 @steveyken @swatijadhav @abulsayyad123 @shreyas098

Reply:
maybe we can check if we can set some flags/settings in that, so that it will look into those things and ignore others.
https://prettier.io/docs/en/options.html